docs/4.12/api-reference/directives.md

```diff
@@ -845,6 +847,35 @@ type Query {
 }
 ```
 
+#### Adding Local Scopes
```

Comment:
Since many directives allow adding scopes, this section might be a better fit in https://lighthouse-php.com/master/eloquent/getting-started.html

Reply:
You are right. I am gonna open a new PR later :)